cuML/src/knn/knn.cu

```diff
@@ -174,9 +174,13 @@ namespace ML {
 
                     try {
                         faiss::gpu::StandardGpuResources gpu_res;
+
+                        cudaStream_t stream;
+                        CUDA_CHECK(cudaStreamCreate(&stream));
```

Comment:
One of the reasons to introduce `cumlHandle` was to control where resources like library handles and CUDA streams are created. Is my assumption correct that this call is still in here because this is the single process multi GPU implementation of knn?

Reply:
I actually didn't update this code but something happened after I rebased the branch.